### PR TITLE
[botkit] Add adapters missing references

### PR DESCRIPTION
**Pull request by LCarbonellRod**
### Description

- Add description to Facebook and Slack packages.

- Add generate package on build on Facebook and Slack packages.

- Add packages references.

### CODE REVIEW

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj [hunk 1]
```diff
@@ -1,9 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <Version Condition=" '$(PreviewPackageVersion)' == '' ">4.6.0-local</Version>
+    <Version Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</Version>
+    <PackageVersion Condition=" '$(PreviewPackageVersion)' == '' ">4.6.0-local</PackageVersion>
+    <PackageVersion Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</PackageVersion>
+    <Configurations>Debug;Release;</Configurations>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
```
Comment (ceciliaavila) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj]: These lines are in the .csproj that was merged to master. Please check if you are using the version in master

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
```
Comment (ceciliaavila) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj]: This is duplicated.

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj [hunk 3]
```diff
@@ -5,10 +5,14 @@
     <PackageVersion Condition=" '$(PackageVersion)' == '' ">4.0.0-local</PackageVersion>
     <PackageVersion Condition=" '$(PackageVersion)' != '' ">$(PackageVersion)</PackageVersion>
     <Configurations>Debug;Release;Debug - NuGet Packages</Configurations>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (ceciliaavila) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj]: This is already in master version. Check if you have the latest version of the adapters.

**Comment by LCarbonellRod** [comment]
> It seems these changes are not necessary. Check master versions of Slack and Facebook adapters.

This changes are already in master, you are right.

**Pull request closed without merging**